src/context-menu.js

```diff
@@ -82,6 +82,15 @@ const TEMPLATE = `<ul id="piling-js-context-menu-list">
   <li>
     <button id="magnify-button">Magnify</button>
   </li>
+  <li>
+    <button id="label-green">Label: green</button>
```

Comment:
Let's remove those predefined buttons. They can be added if needed by the user via custom context menu items.